src/renderer/angular/controllers/commons/dialogs/send-token-controller.js

```diff
@@ -278,6 +283,9 @@ function SendTokenDialogController($rootScope, $scope, $log, $q, $mdDialog, $int
             error = error.toString();
             if (error.indexOf('Insufficient funds') == -1) {
                 CommonService.showToast('error', error, 20000);
+            } else if(error.indexOf('Endpoint request timed out') !== -1){
+                sendEther();
```

Comment:
The `sendToken` and `sendEther` functions look exactly the same to me.

Could we refactor to just use one function for sending both?

Reply:
Yeap, will refactor. Cheers